src/Components/Footer/Footer.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import './Footer.scss'
+
+const Footer = () => (
+    <footer className="footer">
+        <div className="footer_main_box">
+            <div className="footer_first_block">
+                <h2>Lucky</h2>
+                <h5>Community blog</h5>
+                <span>Lorem ipsum dolor sit amet consectetur adipisicing elit. Impedit, pariatur!</span>
+            </div>
+            <div className="footer_second_block">
+                <h3>Follow us</h3>
+            </div>
+        </div>
+        <div className="footer_copyright">
+            <span>Copyright © 2022 community blog. All Rights Reserved</span>
```

Comment:
Please use new Date().getFullYear() to get years, otherwise we will have to edit this code each January 1st :)